frontend/src/scenes/surveys/SurveyView.tsx

```diff
@@ -52,6 +54,13 @@ export function SurveyView({ id }: { id: string }): JSX.Element {
         } else {
             setTabKey('overview')
         }
+        addProductIntent({
```

Comment:
this one can probably be under the `loadSurvey` [loader in the surveyLogic instead](https://github.com/PostHog/posthog/blob/master/frontend/src/scenes/surveys/surveyLogic.tsx#L547)?